nexus/db-queries/src/db/datastore/switch_port.rs

```diff
@@ -1806,37 +1916,36 @@ mod test {
                 geometry: SwitchPortGeometry::Qsfp28x1,
             },
             groups: Vec::new(),
-            links: HashMap::new(),
-            interfaces: HashMap::new(),
-            routes: HashMap::new(),
-            bgp_peers: HashMap::from([(
-                "phy0".into(),
-                BgpPeerConfig {
-                    peers: vec![BgpPeer {
-                        bgp_config: NameOrId::Name(
-                            "test-bgp-config".parse().unwrap(),
-                        ),
-                        interface_name: "qsfp0".into(),
-                        addr: "192.168.1.1".parse().unwrap(),
-                        hold_time: 0,
-                        idle_hold_time: 0,
-                        delay_open: 0,
-                        connect_retry: 0,
-                        keepalive: 0,
-                        remote_asn: None,
-                        min_ttl: None,
-                        md5_auth_key: None,
-                        multi_exit_discriminator: None,
-                        communities: Vec::new(),
-                        local_pref: None,
-                        enforce_first_as: false,
-                        allowed_export: ImportExportPolicy::NoFiltering,
-                        allowed_import: ImportExportPolicy::NoFiltering,
-                        vlan_id: None,
-                    }],
-                },
-            )]),
-            addresses: HashMap::new(),
+            links: vec![],
```

Comment:
Can you update this test to assert full round trip consistency? We're just looking at the number of ports below.

Reply:
The test should be ensuring full round trip consistency now